docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,104 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-Viz
+
+[Plotly](https://plotly.com/python/) is a free and open source Python library that allows you to make interactive, publication quality graphs. With Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation.
+
+Kedro-Viz aims to help users communicate different aspects of their workflow through an interactive flowchart. With Plotly integration, we take one step further in this direction to allow our users to effectively share their data insights while exploring the pipeline.
+
+We have also used the Plotly integration to allow users to [visualise metrics from experiments](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html?highlight=experiment%20tracking).
+
+Note: Kedro's Plotly integration only supports [Plotly Express](https://plotly.com/python/plotly-express/) charts.
```

Comment:
I just realised that users might need to install Plotly Express. Can we just explicitly state that they should do that according to the same methodology we use to [update dependencies](https://kedro.readthedocs.io/en/stable/03_tutorial/04_create_pipelines.html#update-dependencies)?

Reply:
And have you considered @MerelTheisenQB's comment to use RST formatting for notes?

Reply:
hi, since requirements.in is no longer there in starters. I am just going to say you need to add the plotly datasets to requirements.txt